cpp-statsd-client: netinet/in.h for sockaddr_in and IPPROTO_UDP

glibc's arpa/inet.h brings it in, FreeBSD's does not.

--- a/third_party/cpp-statsd-client/include/cpp-statsd-client/UDPSender.hpp	2026-10-28 17:20:46.676485728 +0100
+++ b/third_party/cpp-statsd-client/include/cpp-statsd-client/UDPSender.hpp	2026-10-28 17:20:46.678092713 +0100
@@ -8,6 +8,7 @@
 #include <ws2tcpip.h>
 #else
 #include <arpa/inet.h>
+#include <netinet/in.h>
 #include <netdb.h>
 #include <sys/socket.h>
 #include <unistd.h>
